### PR TITLE
Update Test Apps

### PR DESCRIPTION
**Pull request by guperrot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Added an in-app track check in the test app
[AB#75359](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/75359)

'Update track field' and 'When to update track' = dropdown

Simulator Screen Shot - iphone - 2020-01-28 at 09 59 54

### CODE REVIEW

File: Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift [hunk 1]
```diff
@@ -26,7 +79,12 @@ class MSDistributeViewController: UITableViewController, AppCenterProtocol {
     appCenter.setDistributeEnabled(sender.isOn)
     sender.isOn = appCenter.isDistributeEnabled()
   }
-  
+
+  @IBAction func preStartSwitchUpdated(_ sender: Any) {
+    let startTrackValue = preStartSwitch.isOn ? UpdateTrack.getSelf(by: MSDistribute.updateTrack).state.rawValue : nil
```
Comment (guperrot) [inline Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift]: there is a feedback on other platforms to no use a check box

**Comment by guperrot** [comment]
Recreated from #1948 after rebasing

File: Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift (hunk 1 above)
Comment (clpolet) [reply Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift]: OK let's try to make it consistent between platform if we can. I think this was the original comment: https://github.com/microsoft/appcenter-sdk-android/pull/1354#discussion_r370899599

File: Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift [hunk 2]
```diff
@@ -2,16 +2,93 @@
...
+    case beforeNextStart = "Before next start"
+  }
+
+  private var updatePicker: MSEnumPicker<UpdateTrack>?
+  private var whenUpdateTrackPicker: MSEnumPicker<WhenUpdateTrack>?
+  private var updateTrack = UpdateTrack.Public {
+    didSet {
+       self.updateTrackField.text = self.updateTrack.rawValue
+    }
+  }
```
Comment (guperrot) [inline Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift]:
```suggestion
  }
 
```

**Comment by guperrot** [comment]
Nothing happens when switching "now", I don't know if it's a problem in the base branch.
Private does nothing as well anyway (it believes tester app was opened and nothing happens), probably related.
Cannot validate this for now as the SDK seems to be not working correctly yet.

**Comment by guperrot** [comment]
@ilyashumihin I cannot approve myself as I reopened the PR after the rebase. Looks good to me though for UI behavior. SDK does not work well yet though.

**Comment by guperrot** [comment]
@ilyashumihin for me to approve you can close this PR and reopen it under your name. Sorry for the troubles.

**Pull request merged**